ts/render.ts

```diff
@@ -62,28 +62,43 @@ const ukuleleChords: { [flavor: string]: string }[] = [
 ];
 
 interface InstrumentData {
+  name: string;
+  offset: number;
   strings: number;
   frets: number;
   chords: { [flavor: string]: string }[];
```

Comment:
The `chords` here are unused now.

Reply:
++ I figured I'll delete all the things after the DB is running and stable, including a bunch of *.txt and *.json files from the repo. :o